tarofreighter/chain_porter.go

```diff
@@ -187,6 +187,25 @@ func (p *ChainPorter) RequestShipment(req *AssetParcel) (*PendingParcel, error)
 	}
 }
 
+// RequestSignedDelivery requests a shipment of a virtual transaction that has
+// already been signed. This directly injects the deliverable into the state
+// machine at the state where it is anchored on chain.
+func (p *ChainPorter) RequestSignedDelivery(vPacket *taropsbt.VPacket,
```

Comment:
Very cool that we're able to enable some external interaction with such a small added shim.